app/views/dashboard/index.html.erb

```diff
@@ -2,51 +2,53 @@
 <h1>Dashboard</h1>
 
 <h2>My bookings</h2>
+
 <ul>
   <% @user.bookings.each do |booking| %>
-    <li>
-      Costume: <%= link_to booking.costume.name, costume_path(booking.costume)%> |
-      Start date: <%= booking.start_date %> |
-      End date: <%= booking.end_date %> |
-      Status: <%= booking.status %>
-      <%= link_to "Delete booking", booking_path(booking), method: :delete, data: { confirm: "Are you sure?"} %>
-    </li>
-  <% end %>
-</ul>
-
-
-<!-- case 1 lender false = delete booking
-case 2 lender true & not my costume = delete booking
-case 3 lender true & my costume = edit booking -->
-
-<% if @user.lender %>
-  <!-- <h2>Received bookings</h2>
-  <ul>
-    <% @bookings.each do |booking| %>
+    <% if policy(booking).destroy? %>
       <li>
         Costume: <%= link_to booking.costume.name, costume_path(booking.costume)%> |
         Start date: <%= booking.start_date %> |
         End date: <%= booking.end_date %> |
         Status: <%= booking.status %>
-        <%= link_to "Accept/ reject booking", edit_costume_booking_path(booking.costume, booking) %>
+        <%= link_to "Delete booking", booking_path(booking), method: :delete, data: { confirm: "Are you sure?"} %>
       </li>
     <% end %>
-  </ul> -->
+  <% end %>
+</ul>
 
-  <h2>My costumes</h2>
+
+<h2>Received bookings</h2>
+
+  <ul>
+    <% @received_bookings.each do |booking| %>
+      <% if policy(booking).edit? %>
+        <li>
+          Costume: <%= link_to booking.costume.name, costume_path(booking.costume)%> |
+          Start date: <%= booking.start_date %> |
+          End date: <%= booking.end_date %> |
+          Status: <%= booking.status %>
+          <%= link_to "Accept/ reject booking", edit_costume_booking_path(booking.costume, booking) %>
+        </li>
+      <% end %>
+    <% end %>
+  </ul>
+
+<h2>My costumes</h2>
 
   <ul>
     <% @user.costumes.each do |costume| %>
-      <li>
-        <!-- implement small image + placeholder -->
-        Name: <%= costume.name %> |
-        Size: <%= costume.size %> |
-        Price per day: <%= costume.price_per_day %> |
-        Category: <%= costume.category %> |
-        Description: <%= costume.description %> |
-        <%= link_to "Show", costume_path(costume) %> -
-        <%= link_to "Delete", costume_path(costume), method: :delete, data: { confirm: "Are you sure?" } %>
-      </li>
+      <% if policy(costume).destroy? %>
```

Comment:
I dont think we need `<% if policy(costume).destroy? %>` here anymore, because `@user.costumes` will only display the costumes of the current user, and these costumes can all be deleted by the current user anyway